src/meta/processors/admin/AdminClient.cpp

```diff
@@ -435,6 +435,30 @@ StatusOr<std::vector<HostAddr>> AdminClient::getPeers(GraphSpaceID spaceId, Part
     return Status::Error("Get Failed");
 }
 
+StatusOr<std::vector<HostAddr>> AdminClient::getSpacePeers(GraphSpaceID spaceId) {
```

Comment:
Do we really need it?  I think you could just fanout the request to all active hosts.